LaneStatusMsg lane_status
float64 intersecting_distance
bool directed_right